canonical/ubuntu.com
# Pro change support

## Done

- Make changes to /support for Ubuntu pro release

## QA

- Go to https://ubuntu-com-12078.demos.haus/support 
- Check against the copy: https://docs.google.com/document/d/1AU2dwTXpQk2UuEvSuAqRKrbPWYb-LR0ZGvWu6xw95I8/edit#

## Issue / Card

Fixes #[Issue](https://github.com/canonical/web-squad/issues/6024)
